tests/test_plugin.py

```diff
@@ -3,11 +3,12 @@
 from .common import JSON, Cookies, Headers, Query, Resp, get_paths
 from .test_plugin_falcon import api as falcon_api
 from .test_plugin_flask import api as flask_api
+from .test_plugin_flask import flask_ctx  # noqa: F401
 from .test_plugin_starlette import api as starlette_api
 
 
 @pytest.mark.parametrize("api", [flask_api, falcon_api, starlette_api])
-def test_plugin_spec(api):
+def test_plugin_spec(api, flask_ctx):  # noqa: F811
```

Comment:
No need to add a new `flask_ctx` here. The `spec` property will be cached if you have called it once before.

This can be solved by adding the following code into `tests/test_plugin_flask.py`:

```py
with app.app_context():
    api.spec
```

Reply:
Implemented as suggested.